blueocean-dashboard/src/main/js/OrganizationPipelines.jsx

```diff
@@ -100,25 +105,28 @@ class OrganizationPipelines extends Component {
             delete this.jobListener;
         }
     }
-
-    _selectPipeline(pipelines, params) {
-        if (pipelines && params) {
-            const { pipeline } = params;
-            const p = pipelines.find(aPipeline => aPipeline.fullName === pipeline);
-            if (!p) {
-                return new Failure(`${pipeline} not found`);
-            }
-            return p;
+    
+    _getOrganizationName(nextProps) {
+        if (nextProps && nextProps.params) {
+            return nextProps.params.organization;
+        }
+        if (this.props && this.props.params && this.props.params.organization) {
+            return this.props.params.organization;
+        }
+        if (this.context && this.context.params && this.context.params.organization) {
+            return this.context.params.organization;
```

Comment:
Is this a case where (ugh) destructuring would work? There is lodash "get" (but its stringy, as are many), or other weird suggestions: http://stackoverflow.com/questions/2631001/javascript-test-for-existence-of-nested-object-key